CONTRIBUTING.md

```diff
@@ -1,267 +1,6 @@
-consistent markdown source, and to catch potential formatting issues as early
-as possible. While CI/CD will catch these during the PR review process, you may
-wish to configure your IDE to catch these while writing.
-
-For VSCode, you can use the [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint)
-plugin, which will automatically detect and apply the pre-made rules in
-`.markdownlint.json`. This plugin also comes with auto-fix functionality, see
-the plugin documentation for more information.
-
-## Build and preview the docs locally
-
-On your local machine, clone the docs repository:
-
-```bash
-git clone --recursive https://github.com/docker/docker.github.io.git
-cd docker.github.io
-```
-
-Then, build and run the documentation using [Docker Compose](https://docs.docker.com/compose/)
-
-```bash
-docker compose up -d --build
-```
-
-> You need Docker Compose to build and run the docs locally. Docker Compose is included with [Docker Desktop](https://docs.docker.com/desktop/).
-> If you don't have Docker Desktop installed, follow the [instructions](https://docs.docker.com/compose/install/) to install Docker Compose.
-
-When the container is built and running, visit [http://localhost:4000](http://localhost:4000) in your web browser to view the docs.
-
-To rebuild the docs after you made changes, run the `docker compose up` command
-again. This rebuilds the docs, and updates the container with your changes:
-
-```bash
-docker compose up -d --build
-```
-
-To stop the staging container, use the `docker compose down` command:
-
-```bash
-docker compose down
-```
-
-### Build the docs with deployment features enabled
-
-The default configuration for local builds of the documentation disables some
-features to allow for a shorter build-time. The following options differ between
-local builds, and builds that are deployed to [docs.docker.com](https://docs.docker.com/):
-
-- search auto-completion, and generation of `js/metadata.json`
-- Google Analytics
-- page ratings
-- `sitemap.xml` generation
-- minification of stylesheets (`css/style.css`)
-- adjusting "edit this page" links for content in other repositories
-
-If you want to contribute in these areas, you can perform a "production" build
-locally. To preview the documentation with deployment features enabled, set the `JEKYLL_ENV` environment variable when building the documentation:
-
-```bash
-JEKYLL_ENV=production docker compose up --build
-```
-
-When the container is built and running, visit [http://localhost:4000](http://localhost:4000) in your web browser to view the docs.
-
-To rebuild the docs after you make changes, repeat the steps above.
-
-## Copyright and license
-
-Copyright 2013-2022 Docker, inc, released under the Apache 2.0 license.
+We have created a ['Contribute' section](/contribute/overview.md) in our docs which guides you through the process of contributing to Docker documentation.
```

Comment:
It might have been better to put the full link for contributing overview otherwise contributors will see internal markdown of GitHub with frontmatter stuff and bad formatting:

![image](https://user-images.githubusercontent.com/1951866/188465465-00e521cc-ca80-490a-92a5-df57d579fa06.png)

Smth like:

```
['Contribute' section]({{ site.docs_url }}/contribute/overview.md)
```

Reply:
Oh actually it will not work as GitHub does not handle jekyll syntax in their markdown.

Reply:
Ah, thanks for spotting this Kevin. Have amended in https://github.com/docker/docker.github.io/pull/15567